### DISCUSSION TITLE
Many pipes return this in v0.3.35: Only alphanumeric characters and underscores are allowed in the id

### DISCUSSION THREAD

**Discussion by bogdanr** [open-webui/open-webui#6482] [Q&A] [answered]
## Installation Method

Docker

## Environment

v0.3.35

## Expected Behavior:

I expect to be able to import pipes from the web, such as this one:
https://openwebui.com/f/dotjustin/runware_ai_manifold

## Actual Behavior:

Import fails and I receive this error:
`Only alphanumeric characters and underscores are allowed in the id`

**Comment by dot-Justin** [comment, marked as answer]
Hey, for future reference, this is an issue with the way the title is formatted. To fix it, you should be able to manually edit the title and ID of the function like in the attached GIF.

![1732](https://github.com/user-attachments/assets/9877fc38-71e4-456e-86c1-fc9911f99088)


To prevent this from happening further, I have edited my function and you should be able to just click "Get" and import it without issue. https://openwebui.com/f/dotjustin/runware_flux_manifold/

**Comment by hasanahm-at** [comment]
Hello @dot-Justin , I'm still receving this error when I try to import your function. Many other functions return the same error. Is there a way to fix that?

**Reply by dot-Justin** [reply]
Have you tried changing the Title and ID of the function per my [instructions](https://github.com/open-webui/open-webui/discussions/6482#discussioncomment-11068104)?

**Reply by hasanahm-at** [reply]
Yeah, I'm even trying to import it from the link. It still gives our the same error.

**Reply by dot-Justin** [reply]
That's weird. Would you mind sending a screenshot of the page right after you click submit and get the error?
